### PR TITLE
Updated Docs And Removed Customer Folder

### PR DESCRIPTION
**Pull request by osamaemumba**
Kindly verify that:
1- Except for UsaFacts dataset, the customer folder for all other datasets is deleted.
2- The documentation for **all datasets** is updated:

- Updated table of contents with 'Key Vault Secrets And Access Policies' at 4.
- Parameters list includes key vault name and Azure User Object Id.
- Section 'Key Vault Secrets And Access Policies' (with sub-sections Getting the User Object Id, Add Access Policy and Update Secret Values) is added with all the screenshots.
- All the screenshots appear for the older sections as well.

### CODE REVIEW

File: datasets/jll/USAFacts-data/public/Readme.md [hunk 1]
```diff
@@ -21,16 +22,18 @@
 
    - SQL Database Name
    
-   - SQL Login Administrator Username
+   - SQL Server Administrator Username
    
-   - SQL Login Administrator Password
+   - SQL Server Administrator Password
```
Comment (insidero) [inline datasets/jll/USAFacts-data/public/Readme.md]: Password has been remvoed

Comment (osamaemumba) [reply datasets/jll/USAFacts-data/public/Readme.md]: good point

Comment (osamaemumba) [reply datasets/jll/USAFacts-data/public/Readme.md]: i have removed the password

File: datasets/jll/USAFacts-data/public/Readme.md [hunk 2]
```diff
@@ -145,69 +146,152 @@ If you are using data share to get data from a public environment into the custo
...
 7. Optionally you can enable the snapshot schedule if configured at the public side. For that, checkmark the **Daily** schedule under *Snapshot schedule* tab and click *Enable*.
 
-![data share customer](../customer/images/data%20share/19.png)
+![data share customer](./images/data%20share/19.png)
+
+![data share customer](./images/data%20share/20.png)
+
+## Key Vault Secrets And Access Policies
+
+Azure Key Vault is a cloud service for securely storing and accessing secrets. A secret is anything that you want to tightly control access to, such as API keys, passwords, certificates, or cryptographic keys. Here two secrets are being saved in the key vault with access given to the data factory and to the user whose object id will be provided while creating the deployment (details later):
```
Comment (insidero) [inline datasets/jll/USAFacts-data/public/Readme.md]: **Correction:** Here, two secrets are being saved in the key vault with access given to the data factory and to the user whose object id will be provided while creating the deployment

File: datasets/jll/USAFacts-data/public/Readme.md [hunk 3]
```diff
@@ -145,69 +146,152 @@ If you are using data share to get data from a public environment into the custo
...
+
+1. SQL server administrator password
+2. Connection string for the blob storage 
+
+The user whose object id will be provided is given full access to all secrets, keys and certificates inside the key vault. While the data factory is given GET access to any secret in the key vault.
```
Comment (insidero) [inline datasets/jll/USAFacts-data/public/Readme.md]: **Correction**
While the data factory  has the GET access to the secrets in the key vault.

File: datasets/jll/USAFacts-data/public/Readme.md [hunk 4]
```diff
@@ -145,69 +146,152 @@ If you are using data share to get data from a public environment into the custo
...
+
+### Get the User Object Id
+
+Follow these steps to find a user's object ID in the Azure portal:
+
+1. Type Users in the search bar inside azure portal. Click Users.
```
Comment (insidero) [inline datasets/jll/USAFacts-data/public/Readme.md]: Users should be bold or highlighted

**Review by insidero — CHANGES_REQUESTED** [review]
Kindly make the text bold/italic where it signifies an option or a UI element.

**Pull request merged**